src/js/roadmap/template.js

```diff
@@ -53,13 +47,14 @@ export default function template(json) {
               type="text"
               placeholder="${json.activityPlaceholder}"
             />
-            <button class="clear d-none" id="clearActivity"><span class="ca-gov-icon-close-line" aria-hidden="true"></span> <span class="underline">${json.clearText}</span></button>
+            <button type="button" class="clear d-none" id="clearActivity"><span class="ca-gov-icon-close-line" aria-hidden="true"></span> <span class="underline">${json.clearText}</span></button>
             <ul hidden="" role="listbox" id="awesomplete_list_2"></ul>
+            <div id="activity-error" style="visibility: hidden" class="reopening-field-error text-danger text-small text-left">Activity not found. Search a different term.</div>
```

Comment:
I need to add translations for these errors now. It looked to me like that happens in the Wordpress. Is that right?

Reply:
We send the translations to a professional service after we deploy them in English first.  The json text token combos are here...
https://github.com/cagov/covid19/blob/master/pages/wordpress-posts/safer-economy-lang.json

Reply:
Great! Thank you. I've update this PR. It works as expected in English.

I tried in Spanish to see what would happen without a translation. It looked like it was using some random Spanish words ("borrar", "corrar", etc.) instead of falling back to my English strings. Is that expected?